docs/key-issues/gpus.md

```diff
@@ -30,54 +30,78 @@ Packaging such projects for PyPI has been, and still is, quite challenging.
 
 ## Current state
 
-As of December 2022, PyPI and Python packaging tools are completely unaware of
+As of May 2023, PyPI and Python packaging tools are completely unaware of
 GPUs, and of CUDA. There is no way to mark a package as needing a GPU in sdist
 or wheel metadata, or as containing GPU-specific code (CUDA or otherwise). A
 GPU is hardware that may or may not be present in a machine that a Python
 package is being installed on - `pip` and other installers are unaware of this.
-If wheels contain CUDA code, they require CUDA Toolkit (a specific version of
-it at that) to be installed. Again, installers do not know this and there is no
-way to express this dependency. The same will be true for ROCm and other types
-of GPU hardware and languages.
-
-NVIDIA has made steps towards better support for CUDA on PyPI, through
-CUDA Python ([website](https://developer.nvidia.com/cuda-python), 
-[PyPI package](https://pypi.org/project/cuda-python)), however this is quite
-new and not used by other projects (nor does it target large projects like
-PyTorch and TensorFlow). For most of its own projects, it uses a
-[Private PyPI Index](https://pypi.org/project/nvidia-pyindex/) - and that also
-includes rebuilds of TensorFlow and other packages.
+If wheels contain CUDA code, they require a specific version of the CUDA
+Toolkit (CTK) to be installed. Again, installers do not know this and there is
+no way to express this dependency. The same will be true for ROCm and other
+types of GPU hardware and languages.
+
+NVIDIA has made steps towards better support for CUDA on PyPI. Various
+library components of the CTK have been packaged as wheels and are now
+distributed on PyPI, such as
+[nvidia-cublas-cu11](https://pypi.org/project/nvidia-cublas-cu11/), although
+special care is needed to consume them due to [lack of symlinks in
+wheels](native-dependencies/cpp_deps.md#current-state). Python wrappers around
+CUDA runtime and driver APIs have been consolidated into CUDA Python
+([website](https://developer.nvidia.com/cuda-python), [PyPI
+package](https://pypi.org/project/cuda-python)), but this package assumes that
+the CUDA driver and NVRTC are already installed since it only provides Python
+bindings to the APIs (and no bindings for CUDA libraries are provided as of
+yet). Many other projects remain hosted on [NVIDIA's Private PyPI
+Index](https://pypi.org/project/nvidia-pyindex/), which also includes rebuilds
+of TensorFlow and other packages.
 
 A single CUDA version supports a reasonable range of GPU architectures. New
 CUDA versions get released regularly, and - because they come with increased
 performance or new functionality - it may be necessary or desirable to build
```

Comment:
I know you didn't modify this, but as-is it's hard to parse - how about

```suggestion
CUDA versions get released regularly, and because they come with increased
performance or new functionality it may be necessary or desirable to build
```